css/tabBar.css

```diff
@@ -288,7 +286,6 @@ body.is-focus-mode .navbar-action-button:not(#menu-button) {
 	position: absolute;
 	top: 0;
 	left: 0;
-	margin: 0 0.8em;
```

Comment:
Can you revert this? I think it looks better with the bar aligned to the tab title (as it was before).